operations/beaker/generate_beaker_job

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+template_xml=beaker.tpl.xml
```

Comment:
Maybe we should add the template to the repo. Or at least document that it's required for the script to run. In general, I think it's worth explaining what this script does.